openff/interchange/smirnoff/_nonbonded.py

```diff
@@ -879,6 +897,42 @@ def store_matches(
 
                             # Have this new key (on a duplicate molecule) point to the same potential
                             # as the old key (on a unique/reference molecule)
+                            if type(new_key) is LibraryChargeTopologyKey:
+                                logger.info(
+                                    "Charge section LibraryCharges applied to (topology) atom index "
```

Comment:
```suggestion
                                    "Charge section LibraryCharges applied to topology atom index "
```

I initially read "(topology)" as standing in for some missing data, but I think it's here to disambiguate which atom index is meant? In which case I think it's essential information and doesn't need to be parenthesized